src/download.test.js

```diff
@@ -1,13 +1,31 @@
 const axios = require("axios");
 const tmp = require("tmp");
+jest.mock("axios");
 
 const { SkynetClient, defaultPortalUrl } = require("../index");
+const { trimForwardSlash } = require("./utils_string");
+const { extractNonSkylinkPath } = require("./utils_testing");
```

Comment:
Let's keep the const statements together:

```suggestion
const axios = require("axios");
const tmp = require("tmp");

const { SkynetClient, defaultPortalUrl } = require("../index");
const { trimForwardSlash } = require("./utils_string");
const { extractNonSkylinkPath } = require("./utils_testing");

jest.mock("axios");
```

Reply:
sorry, i can do it